package.json

```diff
@@ -7,15 +7,15 @@
     "test": "test",
     "authlocalinvoke": "sam local invoke IPAuthorizerLambda --event ip-authorizer/event.json --env-vars ip-authorizer/env.json",
     "backlocalinvoke": "sam local invoke BackendLambda --event backend-lambda/event.json --env-vars backend-lambda/env.json",
-    "apilocalinvoke": "sam local start-api --env-vars backend-lambda/env.json"
+    "apilocalinvoke": "sam local start-api --env-vars backend-lambda/env.json",
+    "deploy": "sam build && sam deploy --stack-name im-sam --region us-east-1 --resolve-s3 --confirm-changeset"
```

Comment:
Nope not it. This is kinda confusing but this is the bucket that CF uses to store it's state. We definitely do not want it to be our web app bucket. The commands we need have nothing to do with CF/SAM so the approach of finding new args for this command are not what we want. See the section titled: Bonus: Deploying with AWS CLI in this tutorial: https://wolovim.medium.com/deploying-create-react-app-to-s3-or-cloudfront-48dae4ce0af

Reply:
Here's the CloudFront domain for the react app:

https://d3j9s5m87dlxez.cloudfront.net